pkg/skaffold/schema/latest/v1/config.go

```diff
@@ -517,6 +517,9 @@ type DeployConfig struct {
 
 	// Logs configures how container logs are printed as a result of a deployment.
 	Logs LogsConfig `yaml:"logs,omitempty"`
+
+	// TransformableAllowList configures an allowlist for transforming manifests.
+	TransformableAllowList []ResourceFilter `yaml:"transformableAllowList,omitempty"`
```

Comment:
let's hide this config field until it's used in code

```suggestion
	TransformableAllowList []ResourceFilter `yaml:"-,omitempty"`
```